src/rules/completedDocsRule.ts

```diff
@@ -405,19 +422,27 @@ class CompletedDocsWalker extends Lint.ProgramAwareRuleWalker {
         }
 
         const comments = symbol.getDocumentationComment();
-        this.checkComments(node, nodeType, comments);
+        this.checkComments(node, this.describeNode(nodeType), comments, requirementNode);
+    }
+
+    private describeNode(nodeType: DocType): string {
```

Comment:
Yeah, I thought about some sort of camel-case-to-words thing, but it seemed like overkill. Now that I've changed `enumMembers` to `enum-members`, I could just do:

    return nodeType.replace("-", " ");